CHANGELOG.md

```diff
@@ -72,6 +72,7 @@
 * [ENHANCEMENT] Compactor: Delete bucket-index, markers and debug files if there are no blocks left in the bucket index. This cleanup must be enabled by using `-compactor.no-blocks-file-cleanup-enabled` option. #5648
 * [ENHANCEMENT] Ingester: reduce memory usage of active series tracker. #5665
 * [ENHANCEMENT] Store-gateway: added `-store-gateway.sharding-ring.auto-forget-enabled` configuration parameter to control whether store-gateway auto-forget feature should be enabled or disabled (enabled by default). #5702
+* [ENHANCEMENT] Compactor: added block upload counters `cortex_block_upload_blocks_total`, `cortex_block_upload_bytes_total`, and `cortex_block_upload_files_total`. #5738
```

Comment:
nit: Do you want to mention these are per-tenant?

Reply:
thanks for catching that, I also updated the counter names